1-ES6-practice/madeline-peters/js/app.js

```diff
@@ -14,6 +14,7 @@ const votes = [];
 
 function Product(name) {
   this.name = name;
+  //did not change the next concat because changing the path stopped the site from working
```

Comment:
`img/${name}`.jpg`
This should work. If you're still having problems getting this to work slack me and well figure out whats going wrong